etc/kayobe/kolla/globals.yml

```diff
@@ -10,6 +10,10 @@ enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names }}{% endraw %}"
 # This is necessary for os migrations where mixed clouds might be deployed
 kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}"
 
+# Use facts so this is determined correctly when the control host OS differs
+# from os_distribuition.
+kolla_base_distro_version: "{% raw %}{{ ansible_facts.distribution_major_version }}{% endraw %}"
```

Comment:
Hmm, seems this is wrong:

```
''dict object'' has no attribute ''ubuntu-22'''
```

I think we actually need to do this:
```
kolla_base_distro_version: "{% raw %}{{ kolla_base_distro_version_default_map[kolla_base_distro] }}{% endraw %}"
```

Reply:
Made the change. Tested this does still fix the kolla config diffs. Let's hope CI passes here too now :crossed_fingers: